xml/Microsoft.Extensions.Options/OptionsMonitor`1.xml

```diff
@@ -204,10 +206,12 @@
         <Parameter Name="name" Type="System.String" Index="0" FrameworkAlternate="dotnet-plat-ext-2.0;dotnet-plat-ext-2.1;dotnet-plat-ext-2.2;dotnet-plat-ext-3.0;dotnet-plat-ext-3.1;dotnet-plat-ext-5.0;dotnet-plat-ext-6.0;dotnet-plat-ext-7.0" />
       </Parameters>
       <Docs>
-        <param name="name">To be added.</param>
+        <param name="name">The name of the <typeparamref name="TOptions" /> instance, if <see langword="null" /><see cref="F:Microsoft.Extensions.Options.Options.DefaultName" /> is used.</param>
```

Comment:
```suggestion
        <param name="name">The name of the <typeparamref name="TOptions" /> instance, if a <see langword="null" /> <see cref="F:Microsoft.Extensions.Options.Options.DefaultName" /> is used.</param>
```

Reply:
Thanks for catching. I keep seeing this, so I'm confident it's a bug in the porting tool. I opened https://github.com/dotnet/api-docs-sync/issues/133.